bugsnag-android-core/src/main/java/com/bugsnag/android/SessionTracker.java

```diff
@@ -148,9 +143,7 @@ private void notifySessionStartObserver(Session session) {
             session = new Session(sessionId, date, user, unhandledCount, handledCount);
             notifySessionStartObserver(session);
         } else {
-            setChanged();
-            notifyObservers(new NativeInterface.Message(
-                NativeInterface.MessageType.PAUSE_SESSION, null));
+            notifyObservers(StateEvent.PauseSession.INSTANCE);
```

Comment:
Is register session still used?

Reply:
It's currently used by React Native and Unity - that may change after the latest alterations.